integration_tests/tests/assess/shortNoticeAssessments.cy.ts

```diff
@@ -15,17 +15,15 @@ import AssessHelper from '../../helpers/assess'
 import { addResponseToFormArtifact, addResponsesToFormArtifact } from '../../../server/testutils/addToApplication'
 import { ApprovedPremisesApplication as Application } from '../../../server/@types/shared/models/ApprovedPremisesApplication'
 import applicationDocument from '../../fixtures/applicationDocument.json'
+import { signIn } from '../signIn'
 
 describe('Short notice assessments', () => {
   beforeEach(() => {
     cy.task('reset')
-    cy.task('stubSignIn')
-    cy.task('stubAuthUser')
-  })
 
-  beforeEach(() => {
-    // Given I am logged in
-    cy.signIn()
+    // Given I am signed in as an assessor
+    signIn({ permissions: ['cas1_view_assigned_assessments', 'cas1_assess_application'] })
```

Comment:
I'm wondering if it's worth replicating the roles -> permissions mapping in a separate file. 
i.e. `assessorRole = ['cas1_view_assigned_assessments','cas1_assess_application']`
Then, we could preview any changes of the mapping in the API in one place, rather than having them scattered throughout the integration tests?